starter-code/index.html

```diff
@@ -2,30 +2,45 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+    <link rel="stylesheet" type="text/css" href="css/main.css">
 </head>
 <body>
-  Instagram
-
-  Capture and Share the World's Moments
-
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
-
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-
-  Oh yeah, did we mention it's free?
-
-  Download on the App Store
-  Get it on Google Play
-
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
-
-  &copy; 2014 Instagram
+<div class="page-div">
+  <div class="top-div">
+  <div class = "brand-logo">
+  </div>
+  <a href="#" class="login-btn">Log in</a>
+</div>
+<div class="home-image">
+  <img src="./images/home.png">
```

Comment:
this image should have part of the log in button. It could have been put inside a span.